src/lib/ui/SupportedDeviceTable/SupportedDeviceTable.tsx

```diff
@@ -0,0 +1,14 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+
+import type { SupportedDeviceContentProps } from 'lib/ui/SupportedDeviceTable/SupportedDeviceContent.js'
+import { SupportedDeviceContent } from 'lib/ui/SupportedDeviceTable/SupportedDeviceContent.js'
+
+const client = new QueryClient()
+
```

Comment:
We are exporting props for users for our public components like how mui does it. It's nice to have as a dev :)

```suggestion
export type SupportedDeviceTableProps = SupportedDeviceContentProps

```

Reply:
Oops, forgot this one. Thanks! 👍